cmd/geth/chaincmd.go

```diff
@@ -64,7 +64,7 @@ It expects the genesis file as argument.`,
 		Flags:     utils.NetworkFlags,
 		Category:  "BLOCKCHAIN COMMANDS",
 		Description: `
-The dumpgenesis command dumps the genesis block configuration in JSON format to stdout.`,
```

Comment:
I don't think it makes sense to dump mainnet genesis as the last option. It's very weird we dump out something irrelevant which is not actually requested by users.